packages/shopping/src/application.ts

```diff
@@ -93,7 +92,7 @@ export class ShoppingApplication extends BootMixin(
     this.component(AuthorizationComponent);
 
     // authentication
-    registerAuthenticationStrategy(this, JWTAuthenticationStrategy);
+    this.add(createBindingFromClass(JWTAuthenticationStrategy));
```

Comment:
We should `registerAuthenticationStrategy` if it does not use `createBindingFromClass`.

Reply:
Since it is application level, should the user elect to use the appropriate binding method based on their auth strategy implementation?

Reply:
> We should `registerAuthenticationStrategy` if it does not use `createBindingFromClass`.

Why are we changing this? The comment is missing some words I think...it makes it unclear why we're making this change.

Also, this approach is documented in : https://loopback.io/doc/en/lb4/Loopback-component-authentication.html#registering-a-custom-authentication-strategy .

Reply:
`this.add(createBindingFromClass(JWTAuthenticationStrategy))` can easily be moved into `registerAuthenticationStrategy` for consistency if that is what is desirable. Say the word.

Reply:
@dougal83 The implementation of `registerAuthenticationStrategy` already uses `createBindingFromClass`. What issue did you run into that led to your change?

Reply:
@raymondfeng It was just an arbitrary decision on my part.  Since the strategies were app level, I just shortcut to `this.add(createBindingFromClass(JWTAuthenticationStrategy))`.

I'll alter the code to keep `registerAuthenticationStrategy`.

Reply:
@emonddr Please alter the binding as you see fit going forward.  The documented `registerAuthenticationStrategy` is unchanged and I've shortcut to `this.add(createBindingFromClass(JWTAuthenticationStrategy))` as is in this instance.